### DISCUSSION TITLE
Is it possible to repeat the previous print if it was sent to the printer via OctoPrint?

### DISCUSSION THREAD

**Discussion by DARKFiB3R** [mriscoc/Ender3V2S1#804] [Q&A]
When I go to my "printer cupboard" to remove a print from the bed, it would be nice if there was an option on screen to say "print that again please".

**Comment by dfinberg** [comment]
From Octoprint? You would need to send a command to Octoprint to tell it to send the print file again. You could write an octoprint plugin to watch for specific output and then trigger off that I guess, and then you'd need to add something to the firmware to emit that.

Or you could have octopod on your phone and tap it to print again.

**Reply by DARKFiB3R** [reply]
No, not from Octoprint. From the LCD on the printer.

**Reply by dfinberg** [reply]
Yes, the point is there's no file on the printer to restart (it's all on the pi), and the command structure all goes the other way, from the Pi to the printer.  So you'd need to have octoprint listening for a specific trigger, and then use the LCD to send that trigger.  You could perhaps add a custom menu icon, and then have that emit a string on the console. But it's a bunch of weird workarounds for probably not a lot of gain for most users.

**Comment by TheWebMachine** [comment]
I setup a small touchscreen on my OctoPrint/OctPi servers with the OctoDash interface, which is great for controlling OctoPrint right at the printer. If you are, indeed, using OctoPi on a Pi4 or the like, you can even get a touchscreen enclosure for it to make a nice all-in-one controller akin to Creality's new Pad.

**Reply by DARKFiB3R** [reply]
I am indeed using OctoPi on a Pi4. I have thought about OctoDash but I was hoping it might be possible without it.

**Reply by TheWebMachine** [reply]
Yeah, OctoDash is totally worth it. It was nearly necessary with my Prusa Minis because when connected to OctoPrint, you lose pretty much all control via the local screen and dial. I loved the convenience of OctoDash so much I had an AiO OctoPi/OctoDash screen ready to plug in before my Ender 3 Plus even arrived. 😎
